R/sablefish_om.r

```diff
@@ -3,9 +3,14 @@ rm(list=ls())
 # remotes::install_github('BenWilliams-NOAA/afscOM')
 #library(afscOM)
 library(devtools)
-devtools::load_all("~/Desktop/Projects/afscOM")
+library(patchwork)
+library(tidyverse)
+
+devtools::load_all("~/Desktop/Side projects/afscOM")
+source("~/Desktop/Side projects/SablefishMSE/R/reference_points.R")
+source("~/Desktop/Side projects/SablefishMSE/R/harvest_control_rules.R")
 
-assessment <- dget("~/Desktop/Projects/afscOM/data/test.rdat")
+assessment <- dget("~/Desktop/Side projects/afscOM/data/test.rdat")
```

Comment:
I do this because I forget about it, but let's revert the changed pathing. The `library` calls and `source` calls can stay, but move to to using relative paths when sourcing stuff.